### PR TITLE
Tidy up tox targets

### PR DESCRIPTION
**Pull request by zerolab**
adds Python 3.10 in the mix, as well as Wagtail 2.15 (we already test against main)

### CODE REVIEW

**Comment by zerolab** [comment]
@thibaudcolas any objections to mergin this?

File: tox.ini [hunk 1]
```diff
@@ -2,9 +2,9 @@
 usedevelop = True
 
 envlist =
-    py{36,37,38,39}-dj{22,31}-wagtail{211,212,213}
-    py{36,37,38,39}-dj31-wagtail214
-    py{38,39}-dj{32}-wagtail{213,214}
+    py{37,38,39}-dj{22,31}-wagtail{211,213}
+    py{37,38,39}-dj31-wagtail214
+    py{37,38,39,310}-dj{32}-wagtail{214,215}
```
Comment (thibaudcolas) [inline tox.ini:7]: According to [Wagtail’s compatibility table](https://docs.wagtail.io/en/stable/releases/upgrading.html#compatible-django-python-versions), only Wagtail 2.15 and up support Python 3.10, so I believe this should be:

```suggestion
    py{37,38,39}-dj{32}-wagtail{214,215}
    py{310}-dj{32}-wagtail{215}
```

**Review by thibaudcolas — APPROVED** [review]
@zerolab 👍  I think we might be testing one version too many with Python 3.10, otherwise all good.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -34,6 +28,8 @@ jobs:
         python-version: ["3.6", "3.7", "3.8", "3.9"]
         experimental: [false]
         include:
+          - python-version: "3.10"
+            wagtail: ["2.14", "2.15"]
```
Comment (zerolab) [inline .github/workflows/test.yml:32]:
```suggestion
            wagtail: ["2.15"]
```

File: tox.ini (hunk 1 above)
Comment (zerolab) [reply tox.ini]: Happy to go with the official charts. Thank you @thibaudcolas.
unofficially, it worked well before 2.15 according to the tests :D

**Pull request merged**